doc/QuickStart.md

```diff
@@ -54,14 +82,24 @@ connection.
 Here we clone the data-directory for two PBot instances, naming them after the
 IRC network they will connect to:
 
-    cp -r data freenode
-    cp -r data ircnet
+    $ su pbot
+    $ cd /home/pbot
```

Comment:
As said, local operating system is beyond the scope of this guide. It is up to the users whether they want to create a dedicated user account for pbot or not.